bundle/Controller/Admin/TagController.php

```diff
@@ -45,6 +45,12 @@ public function showTagAction(Request $request, ?Tag $tag = null): Response
         if (!$tag instanceof Tag || !$tag->isSynonym()) {
             $configResolver = $this->getConfigResolver();
 
+            $sortBy = $request->query->get('sort_by');
```

Comment:
I was under the impression that the sorting mechanism will be stored on parent tag, similar to how Ibexa does it in their locations. That way, you don't need to set the sorting for every children fetch, it will automatically fetch tags with the defined sort.

Default sort can then be set in configuration.